internal/operators/cnv/cnv_operator_test.go

```diff
@@ -279,103 +276,114 @@ var _ = Describe("CNV operator", func() {
 				masterWithoutVirt,
 				api.ValidationResult{Status: api.Failure, ValidationId: cnvOperator.GetHostValidationID(), Reasons: []string{"CPU does not have virtualization support"}},
 			),
-			table.Entry("SNO and there is no disk with bigger size than threshold for HPP",
+			Entry("SNO and there is no disk with bigger size than threshold for HPP",
 				&common.Cluster{Cluster: models.Cluster{OpenshiftVersion: "4.10", HighAvailabilityMode: &noneHaMode, Hosts: []*models.Host{masterWithLessDiskSizeAndVirt}}},
 				masterWithLessDiskSizeAndVirt,
 				api.ValidationResult{Status: api.Failure, ValidationId: cnvOperator.GetHostValidationID(), Reasons: []string{"OpenShift Virtualization on SNO requires an additional disk with 53 GB (50 Gi) in order to provide persistent storage for VMs, using hostpath-provisioner"}},
 			),
-			table.Entry("SNO and there is a disk with bigger size than threshold for HPP",
+			Entry("SNO and there is a disk with bigger size than threshold for HPP",
 				&common.Cluster{Cluster: models.Cluster{OpenshiftVersion: "4.10", HighAvailabilityMode: &noneHaMode, Hosts: []*models.Host{masterWithOneSatisfyingDiskAndVirt}}},
 				masterWithOneSatisfyingDiskAndVirt,
 				api.ValidationResult{Status: api.Success, ValidationId: cnvOperator.GetHostValidationID(), Reasons: nil},
 			),
-			table.Entry("Non SNO and there is no disk with bigger size than threshold for HPP shouldn't bother us",
+			Entry("Non SNO and there is no disk with bigger size than threshold for HPP shouldn't bother us",
 				&common.Cluster{Cluster: models.Cluster{OpenshiftVersion: "4.10", HighAvailabilityMode: &fullHaMode, Hosts: []*models.Host{masterWithLessDiskSizeAndVirt}}},
 				masterWithLessDiskSizeAndVirt,
 				api.ValidationResult{Status: api.Success, ValidationId: cnvOperator.GetHostValidationID(), Reasons: nil},
 			),
 		)
 	})
 
-	Context("preflight hardware requirements", func() {
-		fullHaMode := models.ClusterHighAvailabilityModeFull
-		noneHaMode := models.ClusterHighAvailabilityModeNone
-
-		table.DescribeTable("should be returned", func(cfg cnv.Config, cluster common.Cluster) {
-			cnvOperator := cnv.NewCNVOperator(log, cfg)
-			requirements, err := cnvOperator.GetPreflightRequirements(context.TODO(), &cluster)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(requirements.Dependencies).To(ConsistOf(lso.Operator.Name))
-			Expect(requirements.OperatorName).To(BeEquivalentTo(cnv.Operator.Name))
-			numQualitative := 3
-			workerRequirements := newRequirements(cnv.WorkerCPU, cnv.WorkerMemory)
-			masterRequirements := newRequirements(cnv.MasterCPU, cnv.MasterMemory)
-
-			if common.IsSingleNodeCluster(&cluster) {
-				// CNV+SNO installs HPP storage; additional discoverable disk req
-				if cfg.SNOInstallHPP {
-					numQualitative += 1
-				}
-				masterRequirements = newRequirements(cnv.MasterCPU+cnv.WorkerCPU, cnv.MasterMemory+cnv.WorkerMemory)
+	DescribeTable("should be returned", func(cfg cnv.Config, cluster common.Cluster) {
```

Comment:
maybe "preflight hardware requirements should be returned"?
And the change has nothing to do with the commit or the PR:)